nut/Dockerfile

```diff
@@ -8,6 +8,7 @@ COPY rootfs /
 # Setup base
 RUN \
     apk add --no-cache \
+        musl@edge=1.2.1-r0 \
```

Comment:
Meh, this is dangerous as well and unstable as well....
We might be better of building NUT from source maybe?

What do you think?

Reply:
Other option is to switch to Debian I guess.  Building from source will likely be more maintenance (although the source hasn't changed for years).